page/jquery-ui/widget-factory/classes-option.md

```diff
@@ -0,0 +1,134 @@
+<script>{
+	"title": "Using the classes Option",
+	"level": "advanced"
+}</script>
+
+As of the 1.12 release, the jQuery UI widget factory includes a means of managing CSS class names through the [`classes` option](http://api.jqueryui.com/jquery.widget/#option-classes). This article will give you an overview of how the `classes` option works, and discuss what you can do with it.
+
+## Syntax overview
+
+The `classes` option is used to map structural class names to custom class names that you define. To see what this means let's look at an example. The code below uses the `classes` option to create a red dialog:
+
+```
+<style>
+	.custom-red { background: red; }
+</style>
+<script>
+	var dialog = $( "<div>Red</div>" ).dialog({
+		classes: {
+			"ui-dialog": "custom-red"
+		}
+	});
+</script>
+```
+
+Here, the presentational `custom-red` class name is associated with the structural `ui-dialog` class name. Now, whenever the dialog applies the `ui-dialog` class name, it will also add a `custom-red` class name. You can associate multiple class names by including multiple space-delimited class names in the object's value. For instance the following code creates a dialog that is big and red:
+
+```
+<style>
+	.custom-red { background: red; }
+	.custom-big { font-size: 3em; }
+</style>
+<script>
+	var dialog = $( "<div>Big and red</div>" ).dialog({
+		classes: {
+			"ui-dialog": "custom-red custom-big"
+		}
+	});
+</script>
+```
+
+*Note: To get a full list of the class names you can use with the `classes` option, you can log the default values stored at `$.namespace.wigetName.prototype.options.classes`. For instance, the dialog widget's default values are available at `$.ui.dialog.prototype.options.classes`. Additionally, the API documentation for all jQuery UI widgets list the class names they use; here are dialog's: <http://api.jqueryui.com/dialog/#theming>.*
```

Comment:
Ok, I didn't realize the no-value keys were removed. I think it's still worth mentioning the defaults in case people only want to append new class names—e.g. `{ "ui-dialog": $.ui.dialog.prototype.options.classes + " custom-red" }`—but I'll change the wording so that  it's accurate.